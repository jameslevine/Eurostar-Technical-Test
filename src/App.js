import React, { useEffect } from "react";
import { hot } from "react-hot-loader";
import styled from "styled-components";

const App = () => {
  return (
    <Container>
      <Title> Eurostar To-do List </Title>
    </Container>
  );
};

export default hot(module)(App);

const Container = styled.div`
  margin: 1rem;
  font-family: Arial, Helvetica, sans-serif;
`;

const Title = styled.h1`
  text-align: center;
  font-size: 48px;
`;
