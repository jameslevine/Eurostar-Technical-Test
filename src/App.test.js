import App from "./App";
import React from "react";
import { render } from "@testing-library/react";

test("App renders", () => {
  const { getByText } = render(<App />);
  getByText("Eurostar To-do List");
});
