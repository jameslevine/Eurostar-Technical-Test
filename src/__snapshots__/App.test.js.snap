// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My Connected React-Redux Component should render with given state from Redux store 1`] = `
<div
  className="sc-gzVnrw ewKZgt"
>
  <h1
    className="sc-htoDjs htzNlH"
  >
    Eurostar To-do List
  </h1>
  <h2
    className="sc-EHOje cyUrgm"
  >
    200
     results
  </h2>
  <h2
    className="sc-EHOje cyUrgm"
  >
    /
    200
     completed
  </h2>
  <ul
    className="sc-bZQynM hSufOU"
  />
</div>
`;
